testing/src/main/java/org/oppia/android/testing/KonfettiViewMatcher.kt

```diff
@@ -19,9 +19,18 @@ class KonfettiViewMatcher {
     }
 
     /**
-     * A custom [TypeSafeMatcher] class to check that the associated view is a [KonfettiView] and isActive().
+     * This function checks that the view being checked is a [KonfettiView] with the expected number of
```

Comment:
Strike 'This function'. Doc strings should be a fragment that completes the sentence "This method ...".

Reply:
Done